Sources/AvatarView.swift

```diff
@@ -25,33 +25,72 @@
 import Foundation
 
 open class AvatarView: UIView {
-
+    
     // MARK: - Properties
-
+    
+    public var adjustsFontSizeToFitWidth = true
     open var avatar: Avatar = Avatar()
     open var imageView = UIImageView()
+    private var radius: CGFloat?
+    private var font: UIFont? {
+        willSet {
+            if newValue != nil {
+                print("Initialized Font Size: \(String(describing: newValue?.pointSize))")
```

Comment:
Remove this print statement no need to have it clutter up the command line if it working correctly.